relay-server/src/actors/upstream.rs

```diff
@@ -519,6 +519,16 @@ impl UpstreamRelay {
     fn reset_network_error(&mut self) {
         self.first_error = None;
         self.outage_backoff.reset();
+        log::debug!("Recovering from network outage.")
+    }
+
+    fn upstream_connection_check(&mut self, ctx: &mut Context<Self>) {
+        let next_backoff = self.outage_backoff.next_backoff();
+        log::debug!(
+            "Network outage, scheduling another check in {:?}",
```

Comment:
This could even be `warn`, since it is most certainly of interest to consumers of the log stream.

Reply:
Sounds good